localstack-core/localstack/feature_catalog/services/kms_feature.py

```diff
@@ -0,0 +1,195 @@
+from localstack.feature_catalog.service_feature import (
+    ImplementationStatus,
+    ServiceFeature,
+    SupportStatus,
+)
+
+
+class KMSFeature(ServiceFeature):
+    implementation_status: ImplementationStatus = ImplementationStatus.PARTIALLY_IMPLEMENTED
+
+
+# Key Management
+
+
+class KeyManagement(KMSFeature):
+    general_docs: str = "Manages the creation and lifecycle of the cryptographic keys"
+    implementation_status: ImplementationStatus = ImplementationStatus.PARTIALLY_IMPLEMENTED
+    support_type: SupportStatus = SupportStatus.SUPPORTED
+    aws_docs_link: str = (
+        "https://docs.aws.amazon.com/kms/latest/developerguide/getting-started.html"
+    )
+
+
+class Provisioning(KeyManagement):
+    general_docs: str = "Manage the creation and modification of the keys."
+    implementation_status: ImplementationStatus = ImplementationStatus.FULLY_IMPLEMENTED
+    support_type: SupportStatus = SupportStatus.SUPPORTED
+    aws_docs_link: str = "https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html"
+    api_operations: list = ["CreateKey"]
```

Comment:
this is just one suggestion on how to include api_operations.
With the usage as decorators, however, we have shown that we can also link the decorated functions back to the feature-catalog-mapping, which makes this information redundant.